internal/txvalidation/useradmin_tx_validator.go

```diff
@@ -254,18 +261,6 @@ func (v *userAdminTxValidator) validateACLOnUserWrites(operatingUser string, wri
 				ReasonIfInvalid: "the user [" + targetUser + "] is an admin user. Only via a cluster configuration transaction, the [" + targetUser + "] can be modified",
 			}, nil
 		}
-
-		hasPerm, err := v.identityQuerier.HasReadWriteAccessOnTargetUser(operatingUser, targetUser)
-		if err != nil {
-			return nil, err
-		}
-
-		if !hasPerm {
-			return &types.ValidationInfo{
-				Flag:            types.Flag_INVALID_NO_PERMISSION,
-				ReasonIfInvalid: "the user [" + operatingUser + "] has no write permission on the user [" + targetUser + "]",
-			}, nil
-		}
```

Comment:
I think the respective block should be removed also from the `validateACLOnUserDeletes`, otherwise, an admin will not be able to remove a user with ACLs.